wordpress-mobile/WordPress-Android
# Issue 12524/post list fab menu

Fixes #12524
PR description and testing details coming shortly.

## Solution
This solution adds the `Story Post` and `Blog Post` options to the Post List - FAB create button. This functionality even though being merged into `develop` will not be available for release builds.

## Testing
1. Click on the Post List. 
2. Click on the FAB button. 
3. You should now see a bottom sheet with a list of options. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/88587363-77279a80-d01b-11ea-8db0-308fb5f74255.gif" width="320"></kbd>

--------
1. Create a new install of the app.

1. Click on the Post List. 
2. See the "Create a post or story" tooltip for the first launch.

<kbd><img src="https://user-images.githubusercontent.com/1509205/88593403-1d2bd280-d025-11ea-8737-ee98957e436f.png" width="320"></kbd>

## Unit Tests
Unit tests will be created in another PR following this one. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
